src/pages/generate.tsx

```diff
@@ -77,7 +77,7 @@ export default function ClientIdentifierGenerator() {
     logoUri: "",
     tosUri: "",
     policyUri: "",
-    contact: "",
+    contacts: "",
```

Comment:
```suggestion
    contacts: [],
```
The `redirectUris` field should also be initialized to an empty array, rather than an array containing an empty string.